pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2797,4 +2803,7 @@ self: super: {
     ghc-syntax-highlighter = self.ghc-syntax-highlighter_0_0_10_0;
   });
 
+  # Too strict bounds on megaparsec
+  configurator-pg = doJailbreak super.configurator-pg;
```

Comment:
Can you maybe open an upstream issue making them aware of this and link it in a comment.
It would also be nice to have version numbers and a date in the comments so that we can tell when to remove this override.

Reply:
It was already reported and fixed upstream it seems; I added the version and PR as a comment - I assume the jailbreak can be removed with the next Hackage update.